libs/state/src/lib/rx-state.spec.ts

```diff
@@ -95,12 +97,200 @@ describe(rxState, () => {
+        ({ connect }) => {
+          connect('count', counterInput, (state, count) => {
+            return (state?.count ?? count) * count;
+          });
+        },
+        `{{ count() }}`
+      );
+      const state = component.state;
+
+      fixture.detectChanges();
+
+      expect(state.get('count')).toBe(4);
+      expect(fixture.nativeElement.textContent.trim()).toBe('4');
+
+      counterInput.set(4);
+
+      fixture.detectChanges();
+
+      expect(state.get('count')).toBe(16);
+      expect(fixture.nativeElement.textContent.trim()).toBe('16');
+    });
+
+    it('should connect a signal slice', () => {
+      const counterInput = signal({ count: 1337 });
+      const { component, fixture } = setupComponent<{ count: number }>(
+        ({ connect }) => {
+          connect(counterInput);
+        },
+        `{{ count() }}`
+      );
+      const state = component.state;
+
+      fixture.detectChanges();
+
+      expect(state.get('count')).toBe(1337);
+      expect(fixture.nativeElement.textContent.trim()).toBe('1337');
+
+      counterInput.set({ count: 2 });
+
+      fixture.detectChanges();
+
+      expect(state.get('count')).toBe(2);
+      expect(fixture.nativeElement.textContent.trim()).toBe('2');
+    });
+
+    it('should create a computed', () => {
+      const { component } = setupComponent<{
+        count: number;
+        multiplier: number;
+      }>(({ set }) => {
+        set({ count: 1337, multiplier: 1 });
+      });
+      const state = component.state;
+      const multiplied = state.computed(
+        ({ count, multiplier }) => count() * multiplier()
+      );
+
+      expect(multiplied()).toBe(1337);
+
+      state.set({ multiplier: 10 });
+
+      expect(multiplied()).toBe(13370);
+    });
+
+    it('should create a signal using signals and rxjs operators and emit sync without initial value', fakeAsync(() => {
+      const { component } = setupComponent<{
+        count: number;
+        multiplier: number;
+      }>(({ set }) => {
+        set({ count: 1337, multiplier: 1 });
+      });
+      const state = component.state;
+
+      const multiplied = state.computedFrom(
+        pipe(
+          selectSlice(['count', 'multiplier']),
+          map(({ count, multiplier }) => count * multiplier)
+        )
+      );
+
+      expect(multiplied()).toBe(1337);
+      state.set({ multiplier: 10 });
+      expect(multiplied()).toBe(13370);
+    }));
+
+    it('should create a signal using signals and rxjs operators and emit async with startWith', fakeAsync(() => {
+      const { component } = setupComponent<{
+        count: number;
+        multiplier: number;
+      }>(({ set }) => {
+        set({ count: 1337, multiplier: 1 });
+      });
+      const state = component.state;
+
+      const multiplied = state.computedFrom(
+        pipe(
+          selectSlice(['count', 'multiplier']),
+          map(({ count, multiplier }) => count * multiplier),
+          delay(1000),
+          startWith(10)
+        )
```

Comment:
Same question, would be nice to test that we can pass a list of operators instead of using functional pipe.

Reply:
Totally